package.json

```diff
@@ -44,6 +45,7 @@
     "forwarder": "node ./development/static-server.js ./node_modules/@metamask/forwarder/dist/ --port 9010",
     "dapp-forwarder": "concurrently -k -n forwarder,dapp -p '[{time}][{name}]' 'yarn forwarder' 'yarn dapp'",
     "test:unit": "jest",
+    "anvil": "npx anvil",
```

Comment:
This doesn't work on Windows, but `npx anvil.exe` does.  I'm not sure what the best solution for this is, I tried to research it but didn't find anything great.

Reply:
Fixed by using `node_modules/.bin/anvil` directly. Looks like Windows assumes the extension.

Reply:
![image](https://github.com/user-attachments/assets/c44be3f9-9347-4011-9730-fa45c6b3ac62)